[2025-04-21T05:27:41.470+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mbta_ingestion_dag.transform_data scheduled__2025-04-21T05:20:00+00:00 [queued]>
[2025-04-21T05:27:41.477+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mbta_ingestion_dag.transform_data scheduled__2025-04-21T05:20:00+00:00 [queued]>
[2025-04-21T05:27:41.477+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-21T05:27:41.486+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data> on 2025-04-21 05:20:00+00:00
[2025-04-21T05:27:41.491+0000] {standard_task_runner.py:60} INFO - Started process 253 to run task
[2025-04-21T05:27:41.494+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mbta_ingestion_dag', 'transform_data', 'scheduled__2025-04-21T05:20:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/mbta_ingestion_dag.py', '--cfg-path', '/tmp/tmpyjc2khqj']
[2025-04-21T05:27:41.497+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask transform_data
[2025-04-21T05:27:41.509+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-21T05:27:41.539+0000] {task_command.py:423} INFO - Running <TaskInstance: mbta_ingestion_dag.transform_data scheduled__2025-04-21T05:20:00+00:00 [running]> on host 49f544cbb432
[2025-04-21T05:27:41.588+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mbta_ingestion_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T05:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-21T05:20:00+00:00'
[2025-04-21T05:27:41.589+0000] {mbta_ingestion_dag.py:31} INFO - âœ… Transforming data... (dummy placeholder)
[2025-04-21T05:27:41.590+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-21T05:27:41.593+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mbta_ingestion_dag, task_id=transform_data, execution_date=20250421T052000, start_date=20250421T052741, end_date=20250421T052741
[2025-04-21T05:27:41.638+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-21T05:27:41.650+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
